settlement-pipelines/src/bin/claim_settlement.rs

```diff
@@ -61,7 +56,7 @@ struct Args {
         env,
         help = "Path to directory containing json files with tree collection files and settlement files"
     )]
-    merkle_trees_dir: String,
+    merkle_trees_dir: PathBuf,
```

Comment:
Wouldn't list of files be a bit better UX? E.g. running as
```
./claim... --foo bar trees/merkle-*.json
```

Reply:
Agree here.